### PR TITLE
Add e-Money v1.1.3

### PR DESCRIPTION
**Pull request by VitalyVolozhinov**
Adding emoney configuration. This requires this PR to be approved https://github.com/cosmos/chain-registry/pull/55

**NOTE** `run.sh` was edited to ignore `validate-genesis` as emoney doesn't have a `validate-genesis` command.

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -152,7 +152,11 @@ if [ "$DOWNLOAD_GENESIS" == "1" ]; then
 fi
 
 # Validate genesis
-[ "$VALIDATE_GENESIS" == "1" ] && $PROJECT_BIN validate-genesis
+# NOTE: e-money currently doesn't have a `validate-genesis` command.
+if [ $PROJECT_BIN != "emd" ]; then
+  echo "Validating"
+  [ "$VALIDATE_GENESIS" == "1" ] && $PROJECT_BIN validate-genesis
+fi
```
Comment (boz) [inline run.sh:159]: It looks like you can just set this as an environment variable.  See [here](https://github.com/ovrclk/cosmos-omnibus/blob/a73e67fd8c03ec3c3e3f642c5d6b7a0b3f6ae39f/sifchain/deploy.yml#L13)

**Review by boz — COMMENTED** [review]
Thanks @VitalyVolozhinov!  I believe that you don't need to change `run.sh`.  See comment inline.

File: run.sh (hunk 1 above)
Comment (VitalyVolozhinov) [reply run.sh]: Thanks for the help, reverted and added env

**Review by boz — APPROVED** [review]
:rocket:

**Pull request merged**